Ginger/GingerCoreNET/RunLib/CLILib/CLIDynamicXML.cs

```diff
@@ -26,7 +26,7 @@ public class CLIDynamicXML : ICLI
     {
         bool ICLI.IsFileBasedConfig { get { return true; } }
 
-        public string Identifier
+        public string Verb
         {
             get
             {
```

Comment:
should be "return DynamicOptions.Verb;" no?

Reply:
Fixed